client/settings/settings-manager/index.js

```diff
@@ -148,25 +146,49 @@ const SettingsManager = () => {
 
 	useLayoutEffect( () => {
 		const { anchor } = getQuery();
+		const { hash } = window.location;
+		const scrollTo = anchor || hash;
 
-		if ( ! isLoading && anchor ) {
-			document
-				.querySelector( decodeURIComponent( anchor ) )
-				?.scrollIntoView( { behavior: 'smooth' } );
+		if ( ! isLoading && scrollTo ) {
+			const element = document.querySelector( scrollTo );
+
+			if ( ! scrollTo ) {
```

Comment:
Maybe we can check if there's `element` before using it as well? Or we will get an error like this:
<img width="542" alt="error" src="https://user-images.githubusercontent.com/56378160/229903171-4189c0fa-92ba-4996-bb95-627c15e0b1b6.png">


```suggestion
			if ( ! scrollTo || ! element ) {
```

Reply:
I forgot to update this condition, fixed on https://github.com/Automattic/woocommerce-payments/pull/5953/commits/182243ba4b52361d4deea3dd6091e082e31e1172.